tests/installation/ipxe_install.pm

```diff
@@ -148,8 +153,23 @@ sub enter_o3_ipxe_boot_entry {
 }
 
 sub set_bootscript_cmdline_extra {
-
     my $cmdline_extra = " ";
+    if (is_sle('>=16')) {
+        if (get_var('AGAMA_AUTO')) {
+            my $agama_auto = data_url(get_var('AGAMA_AUTO'));
+            $agama_auto =~ s/^\s+|\s+$//g;
+            $cmdline_extra .= "agama.auto=$agama_auto ";
+        }
+        if (get_var('AGAMA_INSTALL_URL')) {
+            my $agama_install_url = get_var('AGAMA_INSTALL_URL');
+            $cmdline_extra .= "agama.install_url=$agama_install_url ";
+        }
+        if (is_ipmi) {
+            my $sol_console = is_aarch64 ? 'ttyAMA0' : 'ttyS1';
```

Comment:
sol console is not fixed for each arch, and for those different , `SERIALDEV` is used to configure, so suggest to use
```
my $sol_console = is_aarch64 ? get_var('SERIALDEV', 'ttyAMA0') : get_var('SERIALDEV', 'ttyS0'); 
```

Reply:
Agree, a minor tip, `get_var('SERIALDEV', 'ttyS0');` => `get_var('SERIALDEV', 'ttyS1');`, because ttyS1 is the default one in other places.

Reply:
will do